### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by LaskoA**

### CODE REVIEW

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
Tests are failed :(

**Comment by LaskoA** [comment]
please check now after Flake8 fixes, thank you.

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,38 @@
...
+
+    def rate_service(self, r):
+        a = self.average_rating * self.count_of_ratings
+        self.average_rating = round((a + r) / (self.count_of_ratings + 1), 1)
+        self.count_of_ratings += 1
+
+    def wash_single_car(self, car):
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+        else:
```
Comment (IvanRamyk) [inline app/main.py:37]: Redundant else branch) 
You don't change anything here

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
+                income += self.calculate_washing_price(item)
+                self.wash_single_car(item)
+        return income
+
+    def calculate_washing_price(self, car):
+        x = car.comfort_class * (self.clean_power - car.clean_mark)
+        y = self.average_rating / self.distance_from_city_center
+        return round(x * y, 1)
+
+    def rate_service(self, r):
```
Comment (IvanRamyk) [inline app/main.py:29]: Please, choose a more descriptive name than `r`

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
The last thing

**Comment by LaskoA** [comment]
Please check, thank you.

**Pull request closed without merging**